### ISSUE TITLE
the NCF wasn't good when I using pre-trained

### ISSUE THREAD

**Issue by Boon-30** [guoyang9/NCF#29]
Can you tell me the exact training parameters when using pre-trained like learning, SGD or Adam, weight decay....? thanks a lot
